test/client/Query.test.tsx

```diff
@@ -524,14 +524,18 @@ describe('Query component', () => {
 
   describe('props allow', () => {
     it('custom fetch-policy', done => {
+      let count = 0;
       const Component = () => (
         <Query query={allPeopleQuery} fetchPolicy={'cache-only'}>
           {result => {
-            catchAsyncError(done, () => {
-              expect(result.loading).toBeFalsy();
-              expect(result.networkStatus).toBe(NetworkStatus.ready);
-              done();
-            });
+            if (count === 0) {
```

Comment:
I had to change this test a bit to get it to pass (due to timing issues in the test only), but it's still testing the same functionality and is accurate.